templates/base.html

```diff
@@ -57,11 +57,12 @@
 										</ul>
 									</li>
 
-									<li><a class="inner-link" href="#speakers" target="default">sponsors</a></li><li><a class="inner-link" href="cfp.html" target="default">call for proposals</a></li><li><a class="inner-link" href="#pricing" target="default">pricing</a></li><li><a class="inner-link" href="#register" target="default">help needed</a></li>
+									<li><a class="inner-link" href="#speakers" target="default">sponsors</a></li><li><a class="inner-link" href="cfp" target="default">call for proposals</a></li><li><a class="inner-link" href="#pricing" target="default">pricing</a></li><li><a class="inner-link" href="#register" target="default">help needed</a></li>
```

Comment:
as this template is reused in numerous pages, I suggest to add absolute URLs in `href` tags